gtk4/src/builder_rust_scope.rs

```diff
@@ -0,0 +1,143 @@
+use crate::subclass::builder_scope::BuilderScopeImpl;
```

Comment:
```suggestion
use crate::subclass::prelude::*;
```

Reply:
Also needs the famous header `// Take a look at the license at the top of the repository in the LICENSE file.`